autogpt_platform/frontend/src/app/library/agents/[id]/page.tsx

```diff
@@ -152,22 +153,20 @@ export default function AgentRunsPage(): React.ReactElement {
         </div>
 
         {/* Run / Schedule views */}
```

Comment:
this ternary is insane -- out of scope though probably?

```suggestion
        {/* Run / Schedule views */}
        {(() => {
          const viewMap = {
            run: selectedView.id
              ? selectedRun && (
                <AgentRunDetailsView
                  agent={agent}
                  run={selectedRun}
                  agentActions={agentActions}
                />
              )
              : (
                <AgentRunDraftView
                  agent={agent}
                  onRun={(runID) => selectRun(runID)}
                  agentActions={agentActions}
                />
              ),
            schedule:
              selectedSchedule && (
                <AgentScheduleDetailsView
                  agent={agent}
                  schedule={selectedSchedule}
                  onForcedRun={(runID) => selectRun(runID)}
                  agentActions={agentActions}
                />
              ),
          };
          return viewMap[selectedView.type] || <p>Loading...</p>;
        })()}
```